featuretools/primitives/standard/aggregation_primitives.py

```diff
@@ -26,7 +26,7 @@ class Count(AggregationPrimitive):
     default_value = 0
 
     def get_function(self):
-        return 'count'
+        return pd.Series.count
```

Comment:
this actually isn't as performant. added question on stackoverflow here: https://stackoverflow.com/questions/55731149/use-a-function-instead-of-string-in-pandas-groupby-agg